cc/sgx/services/fns/sgx_trusted_lib/src/trusted_worker/operators.rs

```diff
@@ -0,0 +1,315 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// Insert std prelude in the top for the sgx feature
+#[cfg(feature = "mesalock_sgx")]
+use std::prelude::v1::*;
+use std::vec;
+use std::iter::FromIterator;
+use std::convert::TryInto;
+
+use crate::worker::{Worker, WorkerContext};
+use crate::trusted_worker::register_func;
+use eigen_core::{Error, ErrorKind, Result};
+use crate::eigen_crypto::sign::ecdsa::KeyPair;
+
+// TODO: unwrap() method should all be replaced with a property error handling
+
+pub struct OperatorWorker {
+    worker_id: u32,
+    func_name: String,
+    input: Option<OperatorWorkerInput>,
+}
+
+impl OperatorWorker {
+    pub fn new() -> Self {
+        OperatorWorker {
+            worker_id: 0,
+            func_name: "operator".to_string(),
+            input: None,
+        }
+    }
+}
+
+enum OperatorKind {
+    Add,
+    Add1,
```

Comment:
xx1 is not easy to understand. How about names like add_cipher_cipher,  add_cipher_plain?